Slight typo in comments of DOM Methods. (#23323)
TheOdinProject/curriculum: foundations/javascript_basics/DOM_manipulation_and_events.md

~~~JavaScript
div.setAttribute('id', 'theDiv');                              

> // if id exists updates it to 'theDiv' else creates an id

Reviewer: ```suggestion
// if id exists, update it to 'theDiv', else create an id
```

// adds class "new" to your new div

div.classList.remove('new');                                   
// removes "new" class from div

div.classList.toggle('active');                                

> // if div doesn't have class "active" then adds it, or if

Reviewer: ```suggestion
// if div doesn't have class "active" then add it, or if
```

> // it does, then removes it

Reviewer: ```suggestion
// it does, then remove it
```
